### ISSUE TITLE
OnTabClosing event for Dragablz TabControl sample

### ISSUE THREAD

**Issue by manfromarce** [Kinnara/ModernWpf#368]
Is there a way to detect and possibly block the closing event of a tab using the TabControl from Dragablz integration sample?

**Comment by Kinnara** [comment]
Yes. The sample's pinned Dragablz 0.0.3.223 package exposes `TabablzControl.ClosingItemCallback`, and its callback args expose `IsCancelled`.

For the sample's named control, a code-behind example is:

```csharp
TabView.ClosingItemCallback = args =>
{
    // Inspect args.DragablzItem / your bound item here.
    if (/* the tab must remain open */)
    {
        args.IsCancelled = true;
    }
};
```

It can also be assigned through XAML/binding, for example `ClosingItemCallback="{Binding ClosingTabItemHandler}"`, where the bound property is an `ItemActionCallback`.

I confirmed those members by inspecting the actual restored 0.0.3.223 assembly used by this repository, then built the integration sample:

`dotnet build .\samples\DragablzSample\DragablzSample.csproj --configuration Release --no-restore`

Result: succeeded with 0 warnings and 0 errors. No ModernWpf product change is needed; the close/cancel hook belongs to Dragablz and is already available.
